great_tables/_tbl_data.py

```diff
@@ -569,12 +569,19 @@ def _(df: PlDataFrame):
     import polars.selectors as cs
 
     list_cols = [
-        name for name, dtype in zip(df.columns, df.dtypes) if issubclass(dtype.base_type(), pl.List)
+        name
+        for name, dtype in df.collect_schema().items()
```

Comment:
If it ever comes to support LazyFrame's as well, this sets up the stage for it :)